### PR TITLE
Add dcm4che-tool-dcmof-2.0.25+imito20191129

### PR DESCRIPTION
**Pull request by DimosN**
DcmOF library now "considers" calling AE Title name (AE Title of requester) as one of the filtering parameters for result worklists (is compared with Dicom tag 'ScheduledStationAETitle' -  (0040,0001)).

### CODE REVIEW

**Comment by rantoniuk** [comment]
SHA checksum files are missing, you might have missed ` -DcreateChecksum=true` option

File: dcm4che/tool/dcm4che-tool-dcmof/2.0.25+imito20191125/dcm4che-tool-dcmof-2.0.25+imito20191125.pom [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dcm4che-tool</artifactId>
+        <groupId>dcm4che.tool</groupId>
+        <version>2.0.25</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>dcm4che.tool</groupId>
+    <artifactId>dcm4che-tool-dcmof</artifactId>
+    <version>2.0.25+imito20191125</version>
```
Comment (rantoniuk) [inline dcm4che/tool/dcm4che-tool-dcmof/2.0.25+imito20191125/dcm4che-tool-dcmof-2.0.25+imito20191125.pom]: Today is 28th, please stick to the current day when PR is opened

**Comment by rantoniuk** [comment]
As today is 20191128, you should be submitting `dcm4che-tool-dcmof-2.0.25+imito20191128` version :)

**Comment by DimosN** [comment]
> SHA checksum files are missing, you might have missed ` -DcreateChecksum=true` option

@warden no, it was added. I tried today again - '_mvn org.apache.maven.plugins:maven-install-plugin:3.0.0-M1:install-file -Dfile=dcm4che-tool-dcmof-2.0.25+imito20191129.jar -DgroupId=dcm4che.tool -DartifactId=dcm4che-tool-dcmof -Dversion=2.0.25+imito20191129 -Dpackaging=jar -DgeneratePom=true -DcreateChecksum=true_' - and still SHA checksum files weren't generated.

**Comment by rantoniuk** [comment]
Whoops, indeed! It is caused by the change in `3.0.0-M1` version:
"The install:install goal does not support creating checksums anymore via -DcreateChecksum=true cause this option has been removed. Details can be found in MINSTALL-143." - https://maven.apache.org/plugins/maven-install-plugin/

Please use `org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file` instead, that generates the checksums correctly, I'll update the README accordingly to use the new version later.

**Comment by DimosN** [comment]
Thank you. Now all necessary files are generated and committed.

**Pull request merged**